sdk/core/Azure.Core/src/Operation.cs

```diff
@@ -16,6 +16,11 @@ namespace Azure
     public abstract class Operation
 #pragma warning restore AZC0012 // Avoid single word type names
     {
+        /// <summary>
+        /// Get a token that can be used to rehydrate the operation.
+        /// </summary>
+        public virtual RehydrationToken? GetRehydrationToken() => null;
```

Comment:
If this is allowed to be null, would it make sense to return null instead of `default(RehydrationToken)` when the service returns `null` JSON?

Reply:
Not following here, the default implementation is returning `null` instead of `default(RehydrationToken)`.